docs/developers/blobstream.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_label: Integrate with Blobstream
+description: Learn how to integrate your L2 with Blobstream
+---
+
+# Integrate with Blobstream
+
+![Blobstream logo](../img/blobstream/blobstream_logo.png)
+
+Blobstream is the first data availability solution for Ethereum that securely
+scales with the number of users. Formerly known as the [Quantum Gravity Bridge (QGB)](https://blog.celestia.org/celestiums/),
+Blobstream relays commitments to Celestia's data root to an onchain light client
+on Ethereum, for integration by developers into L2 contracts. This enables Ethereum
+developers to build high-throughput L2s using Celestia's optimised DA layer,
+the first with Data Availability Sampling (DAS).
+
+The following docs go over how developers can integrate Blobstream. There are also docs on
+how to run a [Blobstream orchestrator](../nodes/blobstream-intro.md) as a Celestia validator
```

Comment:
```suggestion
how to run a [Blobstream orchestrator-relayer](../nodes/blobstream-intro.md) as a Celestia validator
```

or "orchestrator and relayer" -- as there are links on the `blobstream-intro.md` page for both orchestrator and relayer

Reply:
IMO just orchestrator, since validators only run the orchestrator. Relayer is a third-party service.